production/ci/integration/metabackend-start.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+find /app
```

Comment:
This file doesn't do anything

Reply:
I nuked this file. I submitted it by mistake.